.ocamlformat

```diff
@@ -1,32 +1,3 @@
-version=0.26.1
-break-cases = fit
-break-collection-expressions = fit-or-vertical
-break-fun-decl = wrap
-break-fun-sig = wrap
-break-infix = wrap
-break-infix-before-func = false
-break-sequences = false
-break-separators = before
-break-string-literals = never
-break-struct = force
-cases-matching-exp-indent = compact
-doc-comments = after-when-possible
-dock-collection-brackets = false
-indicate-multiline-delimiters = no
-infix-precedence = indent
-let-and = compact
-let-binding-spacing = compact
-module-item-spacing = compact
-parens-tuple = multi-line-only
-parens-tuple-patterns = multi-line-only
-sequence-style = terminator
-sequence-blank-line = compact
-single-case = compact
-type-decl = compact
-if-then-else = keyword-first
-field-space = loose
-space-around-arrays = false
-space-around-records = false
-space-around-lists = false
-space-around-variants = false
-ocp-indent-compat = true
+version=0.27.0
+profile=conventional
+parse-docstrings=false
```

Comment:
parse-docstrings is enabled by default, so explicitly disable it for now...
https://github.com/ocaml-ppx/ocamlformat/blob/b0e0d61ae9262d515b4f16e820d42fd8f2f72d20/lib/Conf.ml#L49-L114